### PR TITLE
[IMP] Accounting: add new version of Accounting/Invoicing initial setup

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: accounting/overview/getting_started/setup.rst [hunk 1]
```diff
@@ -1,36 +1,194 @@
...
-=============================
+===================================================
+Initial setup of Odoo Accounting and Odoo Invoicing
+===================================================
 
-The Odoo Accounting application has an implementation guide that you
-should follow to configure it. It's a step-by-step wizard with links to
-the different screens you will need.
+When you first open your Odoo Accounting app, the *Accounting Overview* page welcomes you with a
+**step-by-step onboarding banner**, a wizard that helps you get started. This onboarding banner
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
step-by-step onboarding banner, a wizard that helps you get started. This onboarding banner
```

File: accounting/overview/getting_started/setup.rst [hunk 2]
```diff
@@ -1,36 +1,194 @@
...
+stays displayed until closed.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
is displayed until you choose to close it.
```

File: accounting/overview/getting_started/setup.rst [hunk 3]
```diff
@@ -1,36 +1,194 @@
...
 
-Once you have `installed the Accounting
-application <https://www.odoo.com/apps/modules/online/account_accountant/>`__,
-you should click on the top-right progressbar to get access to the
-implementation guide.
+All these settings can be modified later by going to :menuselection:`Accounting --> Configuration
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
All these settings being seen in the banner, can be modified later by going to :menuselection:`Accounting --> Configuration
```

I think it means the settings you see in the banner?!

File: accounting/overview/getting_started/setup.rst [hunk 4]
```diff
@@ -1,36 +1,194 @@
...
+#. :ref:`accounting-setup-periods`
+#. :ref:`accounting-setup-chart`
+
+.. _accounting-setup-company:
+
+Company Data
+------------
+
+This menu allows you to add your company’s details such as the name, address, logo, website, phone
+number, email address, and Tax ID, or VAT number.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
number, email address, and Tax ID, or VAT number. These details are then displayed on your documents, such as on invoices.
```

File: accounting/overview/getting_started/setup.rst [hunk 5]
```diff
@@ -1,36 +1,194 @@
...
+
+.. image:: media/setup_company.png
    :align: center
+   :alt: Add your company's details in Odoo Accounting and Odoo Invoicing
+
+These details are then displayed on your documents, such as on invoices.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion

```

Added above.

File: accounting/overview/getting_started/setup.rst [hunk 6]
```diff
@@ -1,36 +1,194 @@
...
+.. note::
+   You can also change these settings by going to :menuselection:`Settings --> General Settings -->
+   Settings --> Companies` and clicking on **Update Info**.
+
+.. _accounting-setup-bank:
+
+Bank Account
+------------
+
+Connect your database to your company’s bank account, and have your bank statements synced
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
Connect your bank account to your company’s database, and have your bank statements synced
```

File: accounting/overview/getting_started/setup.rst [hunk 7]
```diff
@@ -1,36 +1,194 @@
...
-9.  Importing your outstanding transactions
-10. Importing your starting balances
-11. Define the users for accounting
-
-.. image:: ./media/setup02.png
+.. note::
+   :doc:`Click here <../../bank/feeds/bank_synchronization>` for more information about this feature.
+
+If your bank can’t be synchronized automatically, or if you prefer not to sync your bank account
+with your database, you may also configure your bank account manually by clicking on **Create it**,
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
with your database, you may also configure your bank account manually by clicking on *Create it*,
```

File: accounting/overview/getting_started/setup.rst [hunk 8]
```diff
@@ -1,36 +1,194 @@
...
+and filling out the form.
+
+- **Name**: The bank account's name, as displayed on Odoo.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
- **Name**: the bank account's name, as displayed in Odoo.
```

File: accounting/overview/getting_started/setup.rst [hunk 9]
```diff
@@ -1,36 +1,194 @@
...
+- **Account Number**: Your bank account number (IBAN in Europe).
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
- **Account Number**: your bank account number (IBAN in Europe).
```

File: accounting/overview/getting_started/setup.rst [hunk 10]
```diff
@@ -1,36 +1,194 @@
...
+- **Bank**: Click on *Create and Edit* to configure the bank's details. Add the real bank's name
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
- **Bank**: click on *Create and Edit* to configure the bank's details. Add the real bank's name
```

File: accounting/overview/getting_started/setup.rst [hunk 11]
```diff
@@ -1,36 +1,194 @@
...
+  and its Identifier Code (BIC or SWIFT)
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
  and its Identifier Code (BIC or SWIFT).
```

File: accounting/overview/getting_started/setup.rst [hunk 12]
```diff
@@ -1,36 +1,194 @@
...
+- **Code**: This code is your Journal's *Short Code*, as displayed on Odoo. By default, Odoo creates
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
- **Code**: this code is your Journal's *Short Code*, as displayed in Odoo. By default, Odoo creates
```

File: accounting/overview/getting_started/setup.rst [hunk 13]
```diff
@@ -1,36 +1,194 @@
...
+  a new Journal with this Short Code.
+- **Journal**: If this field is displayed, then select the *Journal* you want to use to record the
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
- **Journal**: if this field is displayed, select the *Journal* you want to use to record the
```

File: accounting/overview/getting_started/setup.rst [hunk 14]
```diff
@@ -1,36 +1,194 @@
...
+  financial transactions linked to this bank account or create a new one by clicking on *Create and
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
  financial transactions linked to this bank account, or create a new one by clicking on *Create and
```

File: accounting/overview/getting_started/setup.rst [hunk 15]
```diff
@@ -1,36 +1,194 @@
...
+  Edit*.
+
+.. note::
+   You can add as many bank accounts as needed with this tool by going to :menuselection:`Accounting
+   --> Configuration`, and clicking on **Add a Bank Account**.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
   --> Configuration`, and clicking on *Add a Bank Account*.
```

File: accounting/overview/getting_started/setup.rst [hunk 16]
```diff
@@ -1,36 +1,194 @@
...
+.. _accounting-setup-periods:
+
+Accounting Periods
+------------------
+
+Define here your **Fiscal Years**’ opening and closing dates, which are used to generate reports
+automatically, and your **Tax Return Periodicity**, along with a reminder to never miss a tax return
+deadline.
+
+By default, the opening date is set on the 1st of January and the closing date on the 31st of
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| By default, the opening date is set on the 1st of January and the closing date on the 31st of
```

File: accounting/overview/getting_started/setup.rst [hunk 17]
```diff
@@ -1,36 +1,194 @@
...
+.. note::
+   You can add as many bank accounts as needed with this tool by going to :menuselection:`Accounting
+   --> Configuration`, and clicking on **Add a Bank Account**.
+
+.. _accounting-setup-periods:
+
+Accounting Periods
+------------------
+
+Define here your **Fiscal Years**’ opening and closing dates, which are used to generate reports
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| Define here your **Fiscal Years'** opening and closing dates, which are used to generate reports
```

File: accounting/overview/getting_started/setup.rst [hunk 18]
```diff
@@ -1,36 +1,194 @@
...
+automatically, and your **Tax Return Periodicity**, along with a reminder to never miss a tax return
+deadline.
+
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
```

File: accounting/overview/getting_started/setup.rst [hunk 19]
```diff
@@ -1,36 +1,194 @@
...
+.. note::
+   You can also change these settings by going to :menuselection:`Accounting --> Configuration -->
+   Settings --> Fiscal Periods` and updating the values.
+
+.. _accounting-setup-chart:
+
+Chart of Accounts
+-----------------
+
+With this menu, you can add accounts to your **Chart of Accounts** and indicate their initial
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| With this menu, you can add accounts to your **Chart of Accounts** and indicate their initial
```

File: accounting/overview/getting_started/setup.rst [hunk 20]
```diff
@@ -1,36 +1,194 @@
...
+opening balances.
+
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
```

File: accounting/overview/getting_started/setup.rst [hunk 21]
```diff
@@ -1,36 +1,194 @@
...
+   You can also change these settings by going to :menuselection:`Accounting --> Configuration -->
+   Settings --> Fiscal Periods` and updating the values.
+
+.. _accounting-setup-chart:
+
+Chart of Accounts
+-----------------
+
+With this menu, you can add accounts to your **Chart of Accounts** and indicate their initial
+opening balances.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
  opening balances.
```

as for all the second lines.

File: accounting/overview/getting_started/setup.rst [hunk 22]
```diff
@@ -1,36 +1,194 @@
...
+
+Basic settings are displayed on this page to help you review your Chart of Accounts. To access all
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| Basic settings are displayed on this page to help you review your Chart of Accounts. To access all
```

File: accounting/overview/getting_started/setup.rst [hunk 23]
```diff
@@ -1,36 +1,194 @@
...
+   :alt: Setup of the Chart of Accounts and their opening balances in Odoo Accounting
+
+.. note::
+   :doc:`Click here <chart_of_accounts>` for more information on how to configure your Chart of
+   Accounts.
+
+Invoicing onboarding banner
+===========================
+
+There is another **step-by-step onboarding banner** that helps you take advantage of your Odoo
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| There is another *step-by-step onboarding banner* that helps you to take advantage of your Odoo
```

File: accounting/overview/getting_started/setup.rst [hunk 24]
```diff
@@ -1,36 +1,194 @@
...
+Invoicing and Accounting apps.
+
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
```

File: accounting/overview/getting_started/setup.rst [hunk 25]
```diff
@@ -1,36 +1,194 @@
...
+The **Invoicing onboarding banner** is the one that welcomes you if you use the Invoicing app rather
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| The *Invoicing onboarding banner* is the one that welcomes you if you use the Invoicing app rather
```

File: accounting/overview/getting_started/setup.rst [hunk 26]
```diff
@@ -1,36 +1,194 @@
...
+than the Accounting app.
+
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
```

File: accounting/overview/getting_started/setup.rst [hunk 27]
```diff
@@ -1,36 +1,194 @@
...
+If you have Odoo Accounting installed on your database, you can reach it by going to
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| If you have Odoo Accounting installed on your database, you can reach it by going to
```

File: accounting/overview/getting_started/setup.rst [hunk 28]
```diff
@@ -1,36 +1,194 @@
...
+:menuselection:`Accounting --> Customers --> Invoices`.
+
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
```

File: accounting/overview/getting_started/setup.rst [hunk 29]
```diff
@@ -1,36 +1,194 @@
...
+The Invoicing onboarding banner is composed of four main steps:
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| The Invoicing onboarding banner is composed of four main steps:
```

File: accounting/overview/getting_started/setup.rst [hunk 30]
```diff
@@ -1,36 +1,194 @@
...
+
+This form is the same as :ref:`the one presented in the Accounting onboarding banner
+<accounting-setup-company>`.
+
+.. _invoicing-setup-layout:
+
+Invoice Layout
+--------------
+
+Select here which layout template you want to use for your documents, and with which paper format.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
| Select here which layout template you want to use for your documents, and with which paper format.
```

and check the following paragraphs from here.

File: accounting/overview/getting_started/setup.rst [hunk 31]
```diff
@@ -1,36 +1,194 @@
...
+
+With this tool, you can design the appearance of your documents by selecting a layout, colors, a
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
With this tool, you can design the appearance of your documents by selecting the layout, colors,
```

File: accounting/overview/getting_started/setup.rst [hunk 32]
```diff
@@ -1,36 +1,194 @@
...
+font, and a logo.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
font, and logo.
```

File: accounting/overview/getting_started/setup.rst [hunk 33]
```diff
@@ -1,36 +1,194 @@
...
+
+You can also add your *Company Tagline* and the content of the documents’ *footer*. Note that Odoo
+automatically adds the company's phone number, email, website URL, and Tax ID (or VAT number) to the
+footer, according to values you previously configured in the :ref:`Company Data
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
footer, according to the values you previously configured in the :ref:`Company Data
```

File: accounting/overview/getting_started/setup.rst [hunk 34]
```diff
@@ -1,36 +1,194 @@
...
+   :align: center
+   :alt: Document layout configuration in Odoo Invoicing
+
+.. tip::
+   Add your **bank account number** and a link to your **General Terms & Condition** in the footer.
+   This way, your contacts can find the full content of your GT&C online without having to print
+   them on the invoices you issue.
+
+.. note::
+   These settings can be modified ulteriorly by going to :menuselection:`Settings --> General
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
   These settings can be modified by going to :menuselection:`Settings --> General
```

File: accounting/overview/getting_started/setup.rst [hunk 35]
```diff
@@ -1,36 +1,194 @@
...
+   Settings`, under the *Business Documents* section.
+
+.. _invoicing-setup-payment:
+
+Payment Method
+--------------
+
+This menu helps you configure the payment methods with which your customers can pay you.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
This menu helps you to choose and configure your customer's payment options.
```

File: accounting/overview/getting_started/setup.rst [hunk 36]
```diff
@@ -1,36 +1,194 @@
...
+
+.. important::
+   Configuring a *Payment Acquirer* with this tool also activates the **Invoice Online Payment**
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
   Configuring a *Payment Acquirer* with this tool also activates the *Invoice Online Payment*
```

File: accounting/overview/getting_started/setup.rst [hunk 37]
```diff
@@ -1,36 +1,194 @@
...
+   option automatically. With this, users can directly pay online, from their Customer Portal.
+
+.. _invoicing-setup-sample:
+
+Sample Invoice
+--------------
+
+Send yourself a sample invoice by email and make sure everything is correctly configured.
```
Comment (kbr-odoo) [inline accounting/overview/getting_started/setup.rst]:
```suggestion
Send yourself a sample invoice by email to make sure everything is correctly configured.
```

**Comment by jcs-odoo** [comment]
thanks @kbr-odoo !

**Comment by jcs-odoo** [comment]
@kbr-odoo  indeed, my bad! I didn't load them all.

**Comment by jcs-odoo** [comment]
@kbr-odoo  I couldn't find on Google just right now but I remember being taught not to use line breaks in English the same way I use to do in French.
What I remember is "one idea, one paragraph. no line breaks".

I'll try to find more info about this but for now, I didn't change this in my doc.

Cheers and thanks for the good review :)

File: accounting/overview/getting_started/setup.rst (hunk 25 above)
Comment (kbr-odoo) [reply accounting/overview/getting_started/setup.rst]: I was told to break lines but not paragraphs, and if I'm not mistaken, I was also told that is not necessary to leave blanc lines (I could be wrong here, I don't clearly remember).
-> https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#break-the-line-but-not-the-paragraph

This is my last comment. I think I'm done then!

**Comment by kbr-odoo** [comment]
Example:

| There is another step-by-step onboarding banner that helps you take advantage of your Odoo  
 Invoicing and Accounting apps. The *Invoicing onboarding banner* is the one that welcomes you if 
  you use the Invoicing app rather than the Accounting app.
| If you have Odoo Accounting installed on your database, you can reach it by 
  going to :menuselection:`Accounting --> Customers --> Invoices`.
| The Invoicing onboarding banner is composed of four main steps:

Link with the info (Title: Brek the line but not the paragraph)
https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#break-the-line-but-not-the-paragraph

**Comment by jcs-odoo** [comment]
@kbr-odoo thanks.
It's not about how to do it in RST. I use it often :) but rather to distinguish a "title" from its description (though there is RST code for that too)
It's more about English writing style vs how I write in French.

**Pull request merged**